include/nanorange/algorithm/partial_sort_copy.hpp

```diff
@@ -46,7 +50,7 @@ struct partial_sort_copy_fn {
 
         nano::sort_heap(result_first, r, comp, proj2);
 
-        return r;
+        return {first, r};
```

Comment:
Again, the iterators can be `move`d into the return struct

Reply:
(This is true in lots of places so I won't mark them all)

Reply:
Fixed. I believe I caught all of these cases.